.gitignore

```diff
@@ -85,4 +85,10 @@ build/
 # Mac
 */.DS_Store
 .DS_Store
-.nosync
\ No newline at end of file
+.nosync
+
+app/gui/bin
+app/pkg/bin
```

Comment:
I'm curious where these `bin` directories are coming from.

Reply:
This isn't relevant for this PR, I'll remove.